.vscode/launch.json

```diff
@@ -0,0 +1,134 @@
+{
+    // see https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/debug-launch.md
```

Comment:
I think we could streamline these and increase reliability if we created an install task in `tasks.json`, and then added: `preLaunchTask: "install"` - this way, you'd get a full build  + when you run a launch command? I can try to add an example of this.

Reply:
While it sounds like a good idea, the `install` target takes aronud 30+ secs even w/o code changes due to code signing. Therefore I think it may be better to make this explicit instead of slowing down each debug spawning.

Reply:
CMake tools will trigger a build before launching.